backend/model/model.go

```diff
@@ -1320,6 +1320,7 @@ func MigrateDB(ctx context.Context, DB *gorm.DB) (bool, error) {
 			SELECT project_id, DATE_TRUNC('day', created_at, 'UTC') as date, COUNT(*) as count
 			FROM sessions
 			WHERE excluded <> true
+			AND within_billing_quota
```

Comment:
Just looking at line 1325, is this something we should explicitly check is `true`? Or is that basically what is happening by default?
(Assuming this is the boolean set on the `Session` table)

Reply:
I don't think it makes a difference in the logic, the `excluded <> true` was probably written to include null results but I don't think it does, would probably be cleaner to write as `not excluded` everywhere

```
postgres> select * from (select true as excluded union select false union select null) a where a.excluded <> true
+----------+
| excluded |
|----------|
| False    |
+----------+
```